code/modules/mob/living/carbon/xenomorph/update_icons.dm

```diff
@@ -138,14 +139,47 @@
 
 	wound_overlay.vis_flags &= ~VIS_HIDE // Show the overlay
 
+/mob/living/carbon/xenomorph/proc/update_snowflake_overlays()
```

Comment:
More appropriate name would be `update_backpack_overlay`, also docs

Reply:
https://github.com/tgstation/TerraGov-Marine-Corps/pull/16482#discussion_r1730073650